server/app/lib/fusor/validators/deployment_validator.rb

```diff
@@ -79,6 +80,19 @@ def validate(deployment)
           if !deployment.rhev_is_self_hosted && deployment.rhev_hypervisor_hosts.count < 1
             deployment.errors[:rhev_hypervisor_hosts] << _('RHEV deployments must have at least one Hypervisor')
           end
+
+          special = "?<>',?[]}{=-)(*&^%$#`~{}"
+          regex = /[#{special.gsub(/./) { |char| "\\#{char}" }}]/
+
+          if !(deployment.rhev_engine_host.name =~ regex).nil?
+            add_warning(deployment, _("RHEV engine host do not have proper mac naming scheme."))
+          end
+
+          deployment.rhev_hypervisor_hosts.each do |host|
+            if !(host.name =~ regex).nil?
+              add_warning(deployment, _("RHEV hypervisor hosts do not have proper mac naming scheme."))
```

Comment:
It would be nice to include the hostname in the error message so users know which hostname is invalid.